util/release.rb

```diff
@@ -47,7 +47,7 @@ def create_for_github!
     end
 
     def previous_version
-      @latest_release ||= latest_release.tag_name.gsub(/^#{@tag_prefix}/, "")
+      @previous_version ||= latest_release.tag_name.gsub(/^#{@tag_prefix}/, "")
```

Comment:
Seems additional problem was located thanks to `Naming/MemoizedInstanceVariableName`. :muscle: